packages/ignite-integration-tests/test/ignite-new/newMin.test.js

```diff
@@ -1,79 +0,0 @@
-const test = require('ava')
```

Comment:
I'm super sad to see this integration test go. Can we get a `--max` integration test instead?